torch_glow/src/PyTorchModelLoader.h

```diff
@@ -53,7 +53,7 @@ class PyTorchModelLoader {
         llvm::Error (PyTorchModelLoader::*)(const torch::jit::Node *);
 
     /// Symbols (as strings) that this mapping value is applicable to.
-    const std::vector<const char *> symbols;
+    std::vector<const char *> symbols;
```

Comment:
why this change?

Reply:
By design initialization of the members must be done at construction time, this is self-check making sure no one can change members after initialization.